packages/dashboard/src/features/storage/pages/BucketsPage.tsx

```
@@ -126,10 +126,13 @@ export default function BucketsPage() {
     try {
       setSelectedFiles(new Set());
       setSearchValue('');
-      await Promise.all([
-        refetchBuckets(),
-        queryClient.invalidateQueries({ queryKey: ['storage'] }),
-      ]);
+      const invalidateStorageQueries = [
+        queryClient.invalidateQueries({
+          queryKey: selectedBucket ? ['storage', 'objects', selectedBucket] : ['storage', 'objects'],
```

ESLint

[error] Replace `·?·['storage',·'objects',·selectedBucket]` with `⏎············?·['storage',·'objects',·selectedBucket]⏎···········`

(prettier/prettier)